cloud/deploys/staging_azure/main.tf

```diff
@@ -19,15 +19,23 @@ module "app" {
 
   postgres_admin_login    = var.postgres_admin_login
   postgres_admin_password = var.postgres_admin_password
+  # note that we must use GP tier
+  postgres_sku_name       = "GP_Gen5_2"
 
   docker_username        = var.docker_username
   docker_repository_name = var.docker_repository_name
 
   application_name = var.application_name
   app_secret_key   = var.app_secret_key
   ses_sender_email = var.sender_email_address
+  custom_hostname  = var.custom_hostname
+}
 
-  custom_hostname = var.custom_hostname
+module "custom_hostname" {
+  source              = "../../azure/modules/custom_hostname"
```

Comment:
question (non-blockng): is it possible to use an absolute path from the project base directory? nbd if not

Reply:
as far as i understand it's probably easiest to be local path. if it doesn't start with './ or ../' it assumes it's pulling from a remote repo and we could make a private registry but i think that might be more owrk than it's work https://www.terraform.io/language/modules/sources#modules-in-package-sub-directories

Reply:
ah, no worries then